share/spice/yoga_asanas/yoga_asanas.js

```diff
@@ -0,0 +1,46 @@
+
+function ddg_spice_yoga_asanas(apiResult) {
+
+    if (!apiResult || !apiResult.response.numFound) {
```

Comment:
Is it possible for `apiResult.response` to not exist? Maybe we should check the existence of that too before accessing `numFound`.